docs/architecture/architecture.md

```diff
@@ -0,0 +1,40 @@
+# OpenFGA High Level Architecture
+
+OpenFGA is a server that reads/writes from a database, and its architecture is similar to any other system that does that:
+
+![basic_architecture](deployment.svg)
+
+- The client application is a service that calls OpenFGA. It can call it using the gRPC API or the HTTP API directly, or through OpenFGA's SDKs (JS, Python, Go, Java, .NET). 
```

Comment:
The SDK consume the protocols exposed, this reads as if it was doing something different.